src/ctap/pin_protocol_v1.rs

```diff
@@ -170,6 +172,7 @@ pub struct PinProtocolV1 {
     consecutive_pin_mismatches: u8,
     permissions: u8,
     permissions_rp_id: Option<String>,
+    pub force_pin_change: bool,
 }
 
 impl PinProtocolV1 {
```

Comment:
Do you plan to define a trait for PinProtocol at some point?
By brushing the 2.1 spec, I see they now define also PinProtocolV2.

Reply:
Yes, that is the plan indeed. That was one of the reasons to factor out the PinProtocolV1 implementation, to make it easy to add a PinProtocolV2.